build_tools/bazel_to_cmake/bazel_to_cmake_converter.py

```diff
@@ -277,9 +277,19 @@ def glob(self, include, exclude=None, exclude_directories=1):
         raise NotImplementedError("Recursive globs not supported")
       # Bazel `*.mlir` glob -> CMake Variable `_GLOB_X_MLIR`
       var = "_GLOB_" + pattern.replace("*", "X").replace(".", "_").upper()
-      glob_vars.append(f"${{{var}}}")  # {{ / }} are the escapes for { / }
+      glob_vars.append(var)
       self.converter.body += f"file(GLOB {var} CONFIGURE_DEPENDS {pattern})\n"
-    return glob_vars
+    for pattern in exclude:
+      if "**" in pattern:
+        raise NotImplementedError("Recursive globs not supported")
+      exclude_var = ("_GLOB_" +
+                     pattern.replace("*", "X").replace(".", "_").upper())
+      self.converter.body += (f"file(GLOB {exclude_var} CONFIGURE_DEPENDS "
+                              f"{pattern})\n")
+      for glob_var in glob_vars:
+        self.converter.body += ("list(REMOVE_ITEM " + glob_var + " ${" +
```

Comment:
I decided that boring old string concatenation was more readable than trying to get curly braces with f-strings, which would've been

`f"list(REMOVE_ITEM {glob_var} ${{{exclude_var}}})`

open to differing opinions